Spigot-Server-Patches/0696-Add-ElderGuardianAppearanceEvent.patch

```diff
@@ -0,0 +1,65 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Fri, 19 Mar 2021 23:39:09 -0400
+Subject: [PATCH] Add ElderGuardianAppearanceEvent
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/monster/EntityGuardianElder.java b/src/main/java/net/minecraft/world/entity/monster/EntityGuardianElder.java
+index 9bd6f5d988c1e50e1bb729c407178b9cbcc62115..326cabe208431687be73f7da923037b87f1e6bac 100644
+--- a/src/main/java/net/minecraft/world/entity/monster/EntityGuardianElder.java
++++ b/src/main/java/net/minecraft/world/entity/monster/EntityGuardianElder.java
+@@ -72,6 +72,8 @@ public class EntityGuardianElder extends EntityGuardian {
+             boolean flag1 = true;
+             boolean flag2 = true;
+             boolean flag3 = true;
++            // Paper start
++            if (org.bukkit.craftbukkit.event.CraftEventFactory.handleGuardianAppearanceEvent(this, list)) {
```

Comment:
In my opinion it doesn't make much sense to call this event here, it should be called a bit later when the packet is actually send. That way the event doesn't get fired when no player is affected, and instead of a Player List you would get one Event for each Player